src/app/models/BookManager.java

```diff
@@ -0,0 +1,42 @@
+package models;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+
+public class BookManager {
+
+    private final String bookDirectory;
+
+    public BookManager(String bookDirectory) {
+        this.bookDirectory = bookDirectory;
+    }
+
+    public void listBooks() {
+        File directory = new File(bookDirectory);
+        if(!directory.exists() || !directory.isDirectory()) {
+            System.out.println(Messages.DIR_DOES_NOT_EXIST.getMessage());
+            return;
+        }
+
+        String[] books = directory.list((dir, name) -> name.endsWith(".txt"));
+        if(books == null || books.length == 0){
+            System.out.println(Messages.NO_BOOKS_FOUND.getMessage());
+            return;
+        }
+
+        for(String book : books){
+            System.out.println("Available books found : " + book);
+        }
+
+    }
+
+    public BookAnalyzer getBookAnalyzer(String bookName) throws FileNotFoundException {
```

Comment:
Why do we need this method? We can create BookAnalyzer without this wrapper. Need to be simplified.